shelljs/shelljs.d.ts

```diff
@@ -511,6 +511,18 @@ declare module "shelljs"
      */
     export function error(): string;
 
+
+    export function touch(...files: string[]): void;
+    export function touch(files: string[]): void;
+
+    export function touch(options: string, ...files: string[]): void;
+    export function touch(options: string, files: string[]): void;
+    
+    export function touch(options: Object, ...files: string[]): void;
+    export function touch(options: Object, files: string[]): void;
```

Comment:
Yes, I just pushed that, thank you.
Any recommendation on how to define empty parameters?

```typescript
interface touchOptionsArray = {
   '-a'?: null;
   '-c'?: null; 
}
```